src/divider/__tests__/divider.stories.tsx

```diff
@@ -71,159 +65,147 @@ const myCustomTheme = compileTheme(
   <>
-    <StorybookHeading>Divider - Responsive</StorybookHeading>
-    <StorybookSubHeading>
-      xs: default | sm: vertical | md: horizontal | xs: vertical
-    </StorybookSubHeading>
-    <div style={{height: 50}}>
-      <Divider vertical={{sm: true, md: false, xl: true}} />
-    </div>
+    <StorybookPage>
+      <StorybookCase title="Horizontal XS - Vertical MD - Vertical XL">
+        <div style={{height: 50}}>
+          <Divider vertical={{xs: false, sm: true, md: false, xl: true}} />
+        </div>
+      </StorybookCase>
+    </StorybookPage>
   </>
 );
-StoryResponsive.storyName = 'responsive';
+StoryResponsive.storyName = 'Breakpoint';
 
 export const StoryLogicalPropsOverrides = () => (
   <>
-    <StorybookHeading>Divider - Logical Props</StorybookHeading>
-    <StorybookSubHeading>Logical padding</StorybookSubHeading>
-    <BlockForDivider vertical>
-      <IconFilledFacebook overrides={{size: 'iconSize040'}} />
-      <Divider vertical overrides={{paddingInline: 'space030'}} />
-      <IconFilledWhatsApp overrides={{size: 'iconSize040'}} />
-    </BlockForDivider>
-    <BlockForDivider>
-      <IconFilledFacebook overrides={{size: 'iconSize040'}} />
-      <Divider overrides={{paddingBlock: 'space030'}} />
-      <IconFilledWhatsApp overrides={{size: 'iconSize040'}} />
-    </BlockForDivider>
-    <StorybookSubHeading>Logical margin</StorybookSubHeading>
-    <BlockForDivider vertical>
-      <IconFilledFacebook overrides={{size: 'iconSize040'}} />
-      <Divider vertical overrides={{marginInline: 'space030'}} />
-      <IconFilledWhatsApp overrides={{size: 'iconSize040'}} />
-    </BlockForDivider>
-    <BlockForDivider>
-      <IconFilledFacebook overrides={{size: 'iconSize040'}} />
-      <Divider overrides={{marginBlock: 'space030'}} />
-      <IconFilledWhatsApp overrides={{size: 'iconSize040'}} />
-    </BlockForDivider>
+    <StorybookPage columns={dividerCols}>
+      <StorybookCase title="Padding overrides">
+        <BlockForDivider paddingInline="space030" />
+        <Divider
+          overrides={{paddingBlock: 'space030', paddingInline: 'space030'}}
+        />
+        <BlockForDivider paddingInline="space030" />
+      </StorybookCase>
+      <StorybookCase title="Margin overrides">
+        <BlockForDivider marginInline="space030" />
+        <Divider
+          overrides={{marginBlock: 'space030', marginInline: 'space030'}}
+        />
+        <BlockForDivider marginInline="space030" />
+      </StorybookCase>
+      <StorybookCase title="Styling">
+        <ThemeProvider theme={myCustomTheme}>
+          <Box transparent>
+            <Divider vertical overrides={{stylePreset: 'customDivider'}} />
+          </Box>
+        </ThemeProvider>
+      </StorybookCase>
+    </StorybookPage>
   </>
 );
-StoryLogicalPropsOverrides.storyName = 'logical-props-overrides';
+StoryLogicalPropsOverrides.storyName = 'Overrides';
+
+export default {
+  title: 'Components/Divider',
+  component: 'Divider',
```

Comment:
Can we have this to be the actual component and not a string ?

```suggestion
  component: Divider,
```